subsys/usb/device_next/class/usbd_cdc_acm.c

```diff
@@ -48,6 +48,7 @@ UDC_BUF_POOL_DEFINE(cdc_acm_ep_pool,
 #define CDC_ACM_IRQ_TX_ENABLED		3
 #define CDC_ACM_RX_FIFO_BUSY		4
 #define CDC_ACM_LOCK			5
+#define CDC_ACM_TX_FIFO_BUSY		6
```

Comment:
I do not want to introduce TX_FIFO_BUSY, we can handle it by using delayed work, which is needed anyway, see https://github.com/zephyrproject-rtos/zephyr/pull/75439/commits/741d8a7803c9417f755bbfa98b1f4f6fdf98d668.